up.ps1

```diff
@@ -27,7 +30,13 @@ if ($null -ne $service.Name) {
 }
 
 # Run the docker containers
-docker-compose up -d
+if ($IncludeSugconSites) {
+    Write-Host "Including SUGCON Site containers. Remember to down containers again using file args to bring down all containers." -ForegroundColor Green
+    docker-compose -f docker-compose.yml -f docker-compose.override.yml -f docker-compose.sugcon.yml up -d
+} else {
+    docker-compose up -d
+}
+
```

Comment:
Should we add this same logic to the `docker-compose build` command above?

Reply:
Not needed, the images are all in the override file - nothing to build in the docker-compose..sugcon.yml file

I have another PR coming that separates the init containers into a different compose file too. + the branch with Start-Environment.ps1 (that combines init and up + uses an approved ps1 verb) rather than the 2 separate init/up scripts. So more to come on this.

Reply:
Yeah ok, makes sense.